common/eth2/src/lib.rs

```diff
@@ -1543,6 +1543,28 @@ impl BeaconNodeHttpClient {
         .await
     }
 
+    /// `POST lighthouse/liveness/{epoch}`
+    pub async fn post_lighthouse_liveness_epoch(
```

Comment:
```suggestion
    pub async fn post_validator_liveness_epoch(
```

Similarly this should be renamed and refactored to use the `/eth/v1/validator/liveness` path